src/core/lib/gprpp/memory.h

```diff
@@ -71,7 +71,7 @@ inline UniquePtr<T> MakeUnique(Args&&... args) {
 
 // an allocator that uses gpr_malloc/gpr_free
 template <class T>
-class Allocator {
+class Allocator : public std::allocator<T> {
```

Comment:
Why is this needed?  This is only used in templates, so as long as `Allocator` provides the same API as `std::allocator`, it should not need to actually inherit from it.

Reply:
Your right, I assumed it must inherit to be a type of allocator but the code worked without the inheritance.